### DISCUSSION TITLE
I keep hearing a wolf growl.

### DISCUSSION THREAD

**Discussion by mihai1213** [chromiecraft/chromiecraft#2509] [General]
Hey guys, since I created this new character I keep hearing a wolf growl, like all the time, even tho there is no wolf around me. Do you guys know a fix to this?

**Comment by 55Honey** [comment]
Hi mihai,
what character did you create/in what zone are you?

**Reply by mihai1213** [reply]
Name: Ragnarloot
I am currently in Westfall in Saldean's Farm.

**Reply by mihai1213** [reply]
I hear it every 10 seconds, and it happend everywhere, even in Stormwind.

**Comment by 55Honey** [comment]
That sounds like it was a client-side issue. Please delete the `cache` folder and rename the `WTF` and `Interface` folder in your wow directory.
Let us know how it goes.

**Reply by anthonygauthier** [reply]
I've had that issue as well, whenever I was in an area where wolves are (Elwynn or Dun Morogh) . I'll try your possible solution and let you know how that went!

**Reply by cameron008** [reply]
Well, how did it go? :O
